libraries/AP_Camera/AP_Camera.h

```diff
@@ -69,7 +69,10 @@ class AP_Camera {
 
     // return true if we are using a feedback pin
     bool using_feedback_pin(void) const { return _feedback_pin > 0; }
-    
+
+    // location - get location of gimbal position considering offsets to CG
```

Comment:
Is this CG or IMU origin ?

Reply:
CG is center of gravity. And I think this was menat to be the offset to the IMU origin.
AFAIK all offsets are menat to be relative to the IMU origin.
Or does the IMU also have an offset relative to CG ? In that case all offsets are relative to CG.
@OXINARF any ideas ?

Reply:
The location is AHRS which i thought was CG, but i am not so sure. will change the comment to reflect.

Reply:
This really should be clarified where the origin sits. Since there are offsets for IMU, GPS, and CAM in the parameters, one would assume that the CG or at least an arbitrary point nearby should be the origin.